package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "destructors-ec-bn",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Destructors ecommerce team project",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/index.js",
+    "dev": "nodemon src/index.js",
+    "test": "jest --coverage"
```

Comment:
"test": "jest --coverage --watchAll --verbose", this script can save a lot time, where every time you change a test files you don't need to run npm test, and --verbose can also monitor the history of test